.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,46 @@
+name: "Feature Request"
+description: Suggest an idea for Chroma
+title: "[Feature Request]: "
+labels: ["enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to request this feature!
+  - type: textarea
+    id: problem
+    attributes:
+      label: Describe the problem
+      description: Please provide a clear and concise description the problem this feature would solve. The more information you can provide here, the better.
+      placeholder: I'm always frustrated when...
```

Comment:
Can we change the default language away from frustration?

Reply:
```suggestion
      placeholder: I prefer if...
```